provider/cluster_rosa_classic_resource.go

```diff
@@ -326,12 +415,54 @@ func createClassicClusterObject(ctx context.Context,
 	// ccs should be enabled in ocm rosa clusters
 	ccs := cmv1.NewCCS()
 	ccs.Enabled(true)
+
+	if !state.DisableSCPChecks.Unknown && !state.DisableSCPChecks.Null && state.DisableSCPChecks.Value {
+		ccs.DisableSCPChecks(true)
+	}
 	builder.CCS(ccs)
 
 	aws := cmv1.NewAWS()
+
+	if !state.Tags.Unknown && !state.Tags.Null {
+		tags := map[string]string{}
+		for k, v := range state.Tags.Elems {
+			if _, ok := tags[k]; ok {
+				errDescription := fmt.Sprintf("Invalid tags, user tag keys must be unique, duplicate key '%s' found", k)
+				logger.Error(ctx, errDescription)
+				errHeadline := "Can't build cluster"
```

Comment:
We can declare this variable outside of the `if` scope and then reuse it in line `454`.

Reply:
Could be outside the loop too I suppose